spec.html

```diff
@@ -26381,7 +25846,7 @@ <h1>Object.values ( _O_ )</h1>
       <h1>Properties of the Object Prototype Object</h1>
       <p>The Object prototype object:</p>
       <ul>
-        <li>is <dfn>%ObjectPrototype%</dfn>.</li>
+        <li>is the intrinsic object <dfn>%Object.prototype%</dfn>.</li>
```

Comment:
Why insert "the intrinsic object"? We've mostly eliminated that phrase before `%Foo%` references.

Reply:
oops, you're right